### PR TITLE
Update fhir-package-loader to v0.2.0

### PR DESCRIPTION
**Pull request by cmoesel**
This improves support for "current" dependencies, including support for specific branches of "current" dependencies (via current$mybranch).

### CODE REVIEW

**Comment by cmoesel** [comment]
Whelp. I guess I should have run the tests. Silly me.

File: test/fhirdefs/load.test.ts [hunk 1]
```diff
@@ -46,10 +46,11 @@ describe('#loadCustomResources', () => {
     expect(profiles[2].id).toBe('MyObservation');
     expect(valueSets).toHaveLength(1);
     expect(valueSets[0].id).toBe('MyVS');
-    // Each extension has 3 entries, one for url, one for id, and one for name
-    expect(extensions).toHaveLength(6);
-    const birthPlace = extensions.find(e => e.id === 'patient-birthPlace');
-    const birthPlaceFromXML = extensions.find(e => e.id === 'patient-birthPlaceXML');
+    expect(extensions).toHaveLength(2);
```
Comment (jafeltra) [inline test/fhirdefs/load.test.ts:49]: Oh this is a nice feature from FPL!

**Review by jafeltra — APPROVED** [review]
Makes sense and I confirmed dependencies with a branch can be downloaded now. Yay!

File: test/fhirdefs/load.test.ts (hunk 1 above)
Comment (cmoesel) [reply test/fhirdefs/load.test.ts:49]: Yes, you can thank @mint-thompson for that one! In fact, it looks like [you already did](https://github.com/standardhealth/fhir-package-loader/pull/6#pullrequestreview-992448535)!

**Review by mint-thompson — APPROVED** [review]

**Pull request merged**